clang/include/clang/Format/Format.h

```diff
@@ -3100,6 +3100,54 @@ struct FormatStyle {
   /// \version 11
   TrailingCommaStyle InsertTrailingCommas;
 
+  /// Character case format for different components of a numeric literal.
+  ///
+  /// For all options, ``0`` leave the case unchanged, ``-1``
```

Comment:
nit: `leave` is active tense and `uses` is past-tense.

maybe: `s/leave/leaves`

**OR**: `s/uses/use/g`